packages/redwood/src/index.ts

```diff
@@ -89,46 +70,10 @@ export async function build({
 
   const {
     buildCommand = 'yarn rw db up --no-db-client --auto-approve && yarn rw build',
-    devCommand = 'yarn rw dev',
```

Comment:
I found that `vc dev` already runs the `devCommand` so it was being run twice.

I was able to remove the bulk of the dev logic here besides returning routes.